plugin/clang.vim

```diff
@@ -838,6 +842,52 @@ func! s:ClangCompleteInit(force)
   call s:GlobalVarRestore(l:gvars)
 endf
 "}}}
+"{{{ s:AppendToFile
+" Append line to exist file.
+func! s:AppendToFile(file, lines)
+    call writefile(readfile(a:file)+a:lines, a:file)
+endf
+"}}}
+"{{{ s:JobHandler
+" Handler job events: stdout, stderr, exited.
+func! s:JobHandler(job_id, data, event)
+  if a:event == 'stdout'
+    call s:PDebug("s:JobHandler::event::stdout", a:data, 2)
+    "TODO:
+    let l:datalen = len(a:data)
+    let l:xdata = remove(a:data, l:datalen-1)
+    if filereadable(s:tmps[0])
+      call s:AppendToFile(s:tmps[0], a:data)
+    else
+      call writefile(a:data, s:tmps[0])
+    endif
+  elseif a:event == 'stderr'
+    call s:PDebug("s:JobHandler::event::stderr", a:data, 2)
+    let l:datalen = len(a:data)
+    let l:xdata = remove(a:data, l:datalen-1)
+    if filereadable(s:tmps[1])
+      call s:AppendToFile(s:tmps[1], a:data)
+    else
+      call writefile(a:data, s:tmps[1])
+    endif
+  else
+    call s:PDebug("s:JobHandler::event::exited", a:data, 2)
+    call ClangExecuteDone(s:tmps[0], s:tmps[1])
+  endif
+endf
+"}}}
+"{{{ s:tmps
+" list save tmpfiles name
+let s:tmps = []
```

Comment:
Can we pass tmp file through the `data` arg in job handler ? Global var should be avoid if we can put it locally.